.gitignore

```diff
@@ -12,3 +12,4 @@
 
 # Project-local glide cache, RE: https://github.com/Masterminds/glide/issues/736
 .glide/
+.idea/workspace.xml
```

Comment:
Add full .idea folder to gitignore . No one needs your IDE configs in their local environment)